crates/bvs-vault-cw20-tokenized/src/msg.rs

```diff
@@ -0,0 +1,230 @@
+use bvs_pauser::api::Display;
+use bvs_vault_base::msg::{
+    Amount, AssetsResponse, ConvertToAssetsResponse, ConvertToSharesResponse,
+    QueuedWithdrawalResponse, Recipient, RecipientAmount, SharesResponse, TotalAssetsResponse,
+    TotalSharesResponse, VaultInfoResponse,
+};
+
+use cosmwasm_schema::{cw_serde, QueryResponses};
+use cosmwasm_std::{Binary, Uint128};
+use cw20::Expiration;
+
+#[cw_serde]
+#[derive(Display)]
+pub enum ExecuteMsg {
+    /// Transfer is a base message to move tokens to another account without triggering actions
```

Comment:
```suggestion
    /// ExecuteMsg Transfer is a base message to move tokens to another account without triggering actions
```

Can you also prefix each enum like this:
```rs
/// ExecuteMsg [Name Of Enum]
```

e.g.
```rs
/// ExecuteMsg Transfer
```

If you look at `vault-cw20-tokenized.d.ts` or `schema.go` you can see that the docs are added above the code, instead of at each field.